src/functionalTest/resources/features/F-206 - Request Notice Of Change/S-206.01.td.json

```diff
@@ -7,7 +7,7 @@
   "specs": [
     "Richard - with the ability to create a case for a particular jurisdiction within an organisation",
     "Dil - with a solicitor role for the same jurisdiction, within a different organisation from Richard's",
-    "a citizen Mario, on behalf of whome Richard will create a case",
+    "a citizen Mario, on behalf of whom Richard will create a case",
```

Comment:
Super picky comment, but this could have been a separate PR in order to make this one a bit smaller.